src/docs/user-guide/running-jobs.md

```diff
@@ -25,6 +25,32 @@ Acceptable use of login nodes include:
     CPU time, memory and run time) exceed those limits.
 
 
+## Resource Limits
+
+There are CPU, memory, time and job limits on Sherlock. 
+
+ 
+|   Name |    MaxTRESPU |  MaxSubmitPU | MaxJobsPU |    MaxWall  |   MaxTRESPA  | MaxSubmitPA  
+| ---------- | ------- |  ----------- | --------- | ----------- | ------------- | ----------- |
+normal  |     cpu=256   |     1000   |   |      2-00:00:00 |      cpu=512  |      2000 
+dev | cpu=4,mem=16G   |        2   |       |    02:00:00  |   cpu=99999  |        32 
+long  |      cpu=32     |     20      |  16  | 7-00:00:00      |    |              40 
+bigmem  |      mem=3T     |     10    |   |     1-00:00:00    |    mem=6T    |      20 
+gpu  |  gres/gpu=8     |     50       |   |  2-00:00:00  | gres/gpu=24    |     100
+owner  |  cpu=99999     |   3000       |   |  7-00:00:00   |  cpu=99999     |   5000
+owners  |    cpu=8192     |   3000       |  |   2-00:00:00   |   cpu=8192     |   5000 
+
+For example, on the normal partition a user can use a maximum of 256 CPUs, submit a maximum of 1000 jobs for up to 2 days.  A user's PI acount can run a maximum of 512 CPUs at once and submit up to 2000 jobs.  More information on the values:
```

Comment:
I'd be happy to review this, @kcgthb can give the final go ahead.

 1. I think you might want to say _and_ submit a maximum number of 1000 jobs for up to 2 days.
 2. Also, "user's PI acount" --> "user's PI _account_"
 3. And then instead of "More information on the values" I would say "To see more information on the values:"
 4. For each code block, you should put after the first three "```" the language that it's in. For example (look at the source of this message to see the snippet below is bash)

```bash
$ echo mraaaaaa
```
That ensures that it's colored correctly.